bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -57,13 +58,16 @@
 
 	@Parameter(defaultValue = "${project.build.outputDirectory}", readonly = true)
 	private File classesDir;
-	
+
 	@Parameter(defaultValue = "${project.build.outputDirectory}/META-INF/MANIFEST.MF", readonly = true)
 	private File manifestPath;
-	
+
 	@Parameter(defaultValue = "${project}", required = true, readonly = true)
 	private MavenProject project;
-	
+
+	@Component
+	private BuildContext buildContext;
```

Comment:
So according to this https://wiki.eclipse.org/M2E_compatible_maven_plugins plugins must use buildContext API in order to get incremental workspace build support from m2e.  I assumed that meant the API needed to get injected.  However, let me try without it and if it works I'll remove it.